extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -194,7 +196,20 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 							continue;
 						}
 						try {
-							connectionString = await vscodeMssqlApi.getConnectionString(connectionUri, false, false);
+							// Prompt to include password in connection string
+							if (connectionInfo.password && connectionInfo.authenticationType === constants.basicAuth) {
```

Comment:
1. The auth type shouldn't be localized
2. What happens if the auth type is not basic? We should still be getting the connection string - we just shouldn't be prompting them for the password at that point
3. What happens if the auth type is basic but they don't have the password saved? We should be prompting them to enter it if it isn't (but allowing them to continue if they don't want to)